src/applications/vaos/services/appointment/transformers.js

```diff
@@ -258,6 +258,35 @@ function getRequestedPeriods(appt) {
   return requestedPeriods.sort((a, b) => (a.start < b.start ? -1 : 1));
 }
 
+function setVideoData(appt) {
```

Comment:
This is the core new functionality, transforming into this object instead of the HealthcareService.